### PR TITLE
django 1.8 +, update setup.py, tests, fixed #41 and merged #42, #38

### LINKED ISSUE #41: RemovedInDjango19Warning: 'get_cache' is deprecated in favor of 'caches'
With Django 1.8 I get this warning:

> RemovedInDjango19Warning: 'get_cache' is  deprecated in favor of 'caches'.
>    self.cache = get_cache(self.cache_alias)

It's origin is here: https://github.com/codeinthehole/django-cacheback/blob/master/cacheback/base.py#L79

It should be easy to fix and without the fix, django-cacheback will not work with Django 1.9

### PR DESCRIPTION
**Pull request by michaelkuty**
…r of 'caches'.

self.cache = get_cache(self.cache_alias) last required update for #42

### CODE REVIEW

**Comment by codeinthehole** [comment]
Wow, this looks like great work. Really sorry I've been totally out of touch with my Github repos for so long. Will get this merged and released ASAP.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+    - TOX_ENV=pypy-django18
+    - TOX_ENV=py27-django14
 
 install:
-  - make test
+  - pip install tox
 
 script:
-  - ./runtests.py
+  - tox -e $TOX_ENV
```
Comment (codeinthehole) [inline .travis.yml]: That's neat - I haven't seen that technique for using tox on travis before.

**Comment by codeinthehole** [comment]
All released in v1.0

**Comment by michaelkuty** [comment]
Thanks man.

**Pull request merged**